utils/paste.test.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
There was a lint error `Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks` so I removed `describe('Paste', () => {`